### ISSUE TITLE
[Bug]: latest tag still applied to 1.1.7 in the container registry

### ISSUE THREAD

**Issue by Benjiiim** [Azure/data-api-builder#2358]
### What happened?

It seems that the latest tag is till applied to image 1.1.7 in the Microsoft Artifact Registry instead of the 1.2.10 image.

https://mcr.microsoft.com/en-us/product/azure-databases/data-api-builder/tags

### Version

1.2.10

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Benjiiim** [comment]
cc @Aniruddh25

**Comment by Benjiiim** [comment]
Shouldn't the latest tag be updated when a new image is published in the repository?
I would be very interested to get your thoughts on this @Aniruddh25

**Comment by abhishekkumams** [comment]
That's right @Benjiiim , but currently we are facing some issues with updating the latest tag.

**Comment by abhishekkumams** [comment]
we have updated the tags
